client/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { FiMenu } from "react-icons/fi";
+import { MdClose } from "react-icons/md";
+import styles from "./PageHeader.module.css";
+import { PropTypes } from "prop-types";
+
+const MenuDropdown = ({ onClose }) => {
+  const navigate = useNavigate();
+  const handleLogOut = () => {
+    // TODO : API call for logout
+    navigate("/");
+  };
+
+  return (
+    <div className={styles.navDropdown}>
+      <NavButton hamburger={false} onClick={onClose} />
+      <div className={styles.menuItemContainer}>
+        <MenuDropdownItem
+          label={"Calendar"}
+          action={() => navigate("/calendar")}
+        />
+        <MenuDropdownItem
+          label={"Task History"}
+          action={() => navigate("/task_history")}
+        />
+        <MenuDropdownItem
+          label={"Insights"}
+          action={() => navigate("/insights")}
+        />
+        <MenuDropdownItem label={"Log out"} action={handleLogOut} />
+      </div>
+    </div>
+  );
+};
+
+const MenuDropdownItem = ({ label, action }) => {
+  return (
+    <button className={styles.navDropdownItem} onClick={action}>
```

Comment:
I would also add in hover styles (background changing/text color changing when a user hovers over and `cursor: pointer`
![image](https://user-images.githubusercontent.com/24628243/165216663-bd03a74e-a2f1-460f-a24a-
8979decd65a2.png)

Reply:
Additionally, do we want the menu to be this small? Or perhaps we can make this the entire height of the screen? (Just a suggestion)

![image](https://user-images.githubusercontent.com/24628243/165216795-fb522089-1fba-4690-85cb-dbf44fc75667.png)

Reply:
I have noted down the hover styling to be done together with the rest.

As for the menu, because we have a small number of items on it, I prefer a small dropdown. Initially, I was thinking of a Sidebar too but without the proper animations, I think the large space below it would look awkward.